### DISCUSSION TITLE
Enable bot for the user status change.

### DISCUSSION THREAD

**Discussion by repiatx** [discordjs/discord.js#10092] [Ideas]
Hello folks.

I just wanted a develop bot which can change user status. And i wanted this feature so badly. I even mailed and got declined. :) I kinda understand discord for this but. It is essential for discord to be got in also Business sector.

letme explaein "What this feature holds";
- Changin user status when he/she gets in meeting in every platform ( bot will check and set status )
- Changin user status when he/she writing code
- Changin user status when he/she writes to off channel ( like 20m off )
- and many more

Please conside this option Discord!

Best regards.

**Comment by sdanialraza** [comment]
discord.js is just a wrapper to interact with the API in JavaScript, perhaps you were looking for this? https://github.com/discord/discord-api-docs/discussions/categories/api-feature-requests-ideas

**Comment by repiatx** [comment]
Ouw sorry :/
